vulture_os/services/frontend/models.py

```diff
@@ -748,6 +748,9 @@ class Frontend(models.Model):
         default="",
         verbose_name=_("Cybereason password for authentication")
     )
+    cybereason_timestamp = models.JSONField(
```

Comment:
there is a `last_collected_timestamps` that already holds all generic timestamps, no need to create a new one
see the WAF_BARRACUDA collector for an example